web/modules/custom/collection_page/config/install/field.field.node.collection_page.og_group_ref.yml

```diff
@@ -0,0 +1,29 @@
+langcode: en
```

Comment:
This is as it is originally exported by Drupal and was created by the OG UI.
It works for the time being as a configuration. We cannot actually test it in the scope of this ticket as the only thing this ticket provides is the definition. 
We will test it though during the "Create custom page" ticket.